programs/bench.c

```diff
@@ -317,19 +307,30 @@ BMK_customReturn_t BMK_benchFunction(
         srcSize += srcBlockSizes[ind];
     }
 
+    {
+        size_t i;
+        for(i = 0; i < blockCount; i++) {
+            memset(dstBlockBuffers[i], 0xE5, dstBlockCapacitiesToSizes[i]);  /* warm up and erase result buffer */
+        }
+
+        //UTIL_sleepMilli(5);  /* give processor time to other processes */
```

Comment:
explain why you believe it's better to disable these lines,
prefer using preprocessor directives, such as `#if 0`